install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p ~/Library/Developer/Xcode/Templates/File\ Templates
+mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates
+mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates/Nimble\ Templates
```

Comment:
what is the following for? If we need more files, we can make sub folders inside our template.
```
mkdir -p ~/Library/Developer/Xcode/Templates/File\ Templates
 mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates
```

Reply:
I just followed from Apple's Templates, in Templates folder from Xcode, it has `File Templates` and `Project Templates` folders. So if we need more files, I think we will add file template into `File Templates/Nimble Templates`. What do you think?

Reply:
We could create `~/Library/Developer/Xcode/Templates/Nimble\ Templates` and have subfolders inside that?

Reply:
If we create a subfolder (e.g `File Templates` or `Project Templates`), the name of section when create new project or file is not showing in the right name.

<img width="735" alt="Screenshot_29_9_20_12_12" src="https://user-images.githubusercontent.com/19943832/94515119-1439c600-024d-11eb-91da-f376e32a8957.png">

![ContentView_swift](https://user-images.githubusercontent.com/19943832/94515035-e3f22780-024c-11eb-93ee-e09e1b038f5c.png)

So I suggest: 
```
~/Library/Developer/Xcode/Templates/File Templates/Nimble Templates
~/Library/Developer/Xcode/Templates/Project Templates/Nimble Templates
```

So we can separate file and project templates into 2 folders same like Apple did

<img width="1025" alt="Screen Shot 2020-09-29 at 12 14 35" src="https://user-images.githubusercontent.com/19943832/94515261-59f68e80-024d-11eb-8b17-ed5cb1447357.png">

What do you think?